code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -465,3 +476,94 @@ GLOBAL_LIST_INIT(abductor_recipes, list ( \
 
 /obj/item/stack/sheet/mineral/coal/ten
 	amount = 10
+
+/*
+ * Morphium
+ */
+
+/obj/item/stack/sheet/mineral/morphium
+	name = "Morphium"
+	desc = "A powerful material that mimics what the gray* use."
+	icon_state = "sheet-wavy"
+	item_state = "mhydrogen"
+	icon_colour = "#37115A"
+
+/*
+ * Valhollide
+ */
+
+/obj/item/stack/sheet/mineral/valhollide
+	name = "Valhollide"
+	desc = "A powerful material made out of mashing particles into one another, ya science!"
```

Comment:
You're gunna have to link me this reference, lils, because this just looks like a word salad to me.

Reply:
ya science! - Braking bad right?